schema/2.0.0/corpora_schema.md

```diff
@@ -670,6 +674,26 @@ When a dataset is uploaded, the cellxgene Data Portal MUST automatically add the
 </tbody></table>
 <br>
 
+### feature_reference
+
+<table><tbody>
+    <tr>
+      <th>Key</th>
+      <td>feature_reference</td>
+    </tr>
+    <tr>
+      <th>Annotator</th>
+      <td>Data Portal</td>
```

Comment:
Doesn't the curator set this?

Reply:
I added changes to address this

Reply:
I might have added a _for example_, but I could add that in a later editorial update. I've seen a few things that I could address ... and I want to add a STRONGLY RECOMMENDED for unique titles for datasets per a side conversation with Jason.